### DISCUSSION TITLE
RBAC provider that uses Kafka ACLs

### DISCUSSION THREAD

**Discussion by craiglservin** [kafbat/kafka-ui#482] [Tech Proposals]
By combining an Oauth2 Identity provider with the existing Topic ACLs and using the identity from Oauth to match the subject of the ACLs it would allow for a very easy rollout of kafka-ui without having to produce/update any special RBAC configuration file.

**Comment by Haarolean** [comment]
Created an issue rather than a discussion
https://github.com/kafbat/kafka-ui/issues/484
